internal/runners/clean/cache.go

```diff
@@ -44,6 +55,27 @@ func (c *Cache) Run(params *CacheParams) error {
 		}
 	}
 
-	logging.Debug("Removing cache path: %s", c.path)
+	logging.Debug("Removing cache path: %s", path)
 	return removeCache(c.path)
 }
+
+func (c *Cache) removeProject(namespace string, force bool) error {
+	if !force {
+		ok, fail := c.confirm.Confirm(locale.Tr("clean_cache_artifact_confirm", namespace), false)
+		if fail != nil {
+			return fail.ToError()
+		}
+		if !ok {
+			return nil
+		}
+	}
+
+	split := strings.Split(namespace, "/")
```

Comment:
http://github.com/ActiveState/cli/blob/c06b9f831c3b6bddb2a27fd50e9e4539df3895d1/pkg/project/namespace.go#L71-L71

Reply:
I even looked for that function! 🤦